src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/ConfigReader.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.deciders.Decider;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ConfigReader {
```

Comment:
nit: rename to something specific to action priorities?

Reply:
I think we should limit this one to only action priorities, to keep it simple to read. Configuration values could go into rca.conf along with other config values. But you can take a call on that if you think there are other decider level configs that should belong here. 
Also, if you want to make it generic you (at least) need another level of hierarchy in the yml, right?